### ISSUE TITLE
Productivity Tracker App 

### ISSUE THREAD

**Issue by sanjana-coder** [Vishal-raj-1/Awesome-JavaScript-Projects#248]
We all have come across **TODO Apps** which keeps track of tasks which we are supposed to do in the near future. Once the task is complete it is **struck off** the list.
Nowadays people find it difficult to keep track of what all they have done!
This app aims to keep track of the tasks the person has completed in the day and also provide notifications what help increase productivity.
**Features** 
- Keep count of tasks completed
- Productivity Tips
- Summary of how much time is spent productively

In short this is a **Productivity Tracker App**! 

I would like to contribute this project to your site. I'll give it my best shot!

Kindly assign this as an issue to me @urvashi-code1255 @Vishal-raj-1, with **GSSoC21** label
Thank you

**Comment by urvashi-code1255** [comment]
@sanjana-coder Please go for it :)

**Comment by sanjana-coder** [comment]
Thank you!!

**Comment by sanjana-coder** [comment]
Could you please add gssoc21 label to the issue
@urvashi-code1255 @Vishal-raj-1 
Thank you

**Comment by sanjana-coder** [comment]
Please add gssoc21 label to the issue
@urvashi-code1255 @Vishal-raj-1
Thank you

**Comment by aliya-rahmani** [comment]
@urvashi-code1255 can you also assign me this task?
